packages/component-library/src/Sandbox/SandboxBaseMapSelector.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { css } from "emotion";
+
+const baseMapMenu = css(`
+  position: absolute;
+  background: #f3f2f3;
+  padding: 10px;
+  bottom: 0;
+  right: 0;
+  z-index: 2;
+  font-family: "Roboto Condensed", sans-serif;
+`);
+
+const SandboxBaseMapSelector = ({ onBaseMapStyleChange, baseMapStyle }) => {
+  return (
+    <>
```

Comment:
Might consider using `<React.Fragment></React.Fragment>` here to get rid of the warning.